### PR TITLE
fix(FEC-11536):expose missing fields to adInfo object

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

expose missing fields to adIOptions object

solves: FEC-11536

related pr:
https://github.com/kaltura/kaltura-player-js/pull/496
https://github.com/kaltura/playkit-js/pull/608

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/ima-state-machine.js [hunk 1]
```diff
@@ -540,6 +540,9 @@ function getAdOptions(adEvent: any): Object {
   adOptions.bitrate = ad.getVastMediaBitrate();
   adOptions.bumper = podInfo.getIsBumper() || this._isBumper;
   adOptions.vpaid = this._isVpaid;
+  adOptions.wrapperAdIds = ad.getWrapperAdIds();
+  adOptions.wrapperCreativeIds = ad.getWrapperCreativeIds();
+  adOptions.wrapperAdSystems = ad.getWrapperAdSystems();
```
Comment (OrenMe) [inline src/ima-state-machine.js:545]: this is breaking the adOptions object that is defined in kaltura-player-js and the `PKAdOptions` type

**Review by OrenMe — CHANGES_REQUESTED** [review]

File: src/ima-state-machine.js (hunk 1 above)
Comment (OrenMe) [reply src/ima-state-machine.js:545]: need to use the object type in IMA plugin

Comment (JonathanTGold) [reply src/ima-state-machine.js:545]: i updated the type in the playkit and the in the Ad class in kalture-player.js too
https://github.com/kaltura/kaltura-player-js/pull/496
https://github.com/kaltura/playkit-js/pull/608

**Comment by kaltura-hooks** [comment]
Hi @JonathanTGold,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Review by yairans — APPROVED** [review]

**Pull request merged**